docs/io.cozy.photos.md

```diff
@@ -22,9 +22,8 @@ This doctype is used to store settings for photos, useful for the clustering.
 
 ### Clustering settings
 
-- `lastDate`: {date} the date of the last clustered photo (based on `created_at`).
+- `lastDate`: {date} the date of the last clustered photo (based on `cozyMetadata.createdAt`).
```

Comment:
based on `cozyMetadata.createdAt` of what? Of the last clustered photo io.cozy.file document?

Reply:
Yes, is there a possible confusion? I don't see what else it could be :thinking:

Reply:
I rephrased to remove any possible confusion ;)

Reply:
Me neither, but it was to be 100% sure.